dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -900,6 +938,44 @@
                     "REOPEN"
                   ]
                 },
+                "specialIssues": {
+                  "type": "array",
+                  "maxItems": 100,
+                  "items": {
+                    "type": "object",
+                    "required": [
+                      "code",
+                      "name"
```

Comment:
EVSS's `specialIssue` field is a `String` in the swagger instead of a sub-object containing `name` and `code`.

Reply:
Not sure how it would change for the schema. Per the payload, code and name fields are of type `String` and are placed under `specialIssues` array

Reply:
@SMLuthi @cvalarida : `code` and `name` has been removed and now matches the v2 version for all claims